test/core/translate/Translate_test.cpp

```diff
@@ -3,11 +3,19 @@
 #include "include/AtlasTypes.hpp"
 #include "core/PageTableEntry.hpp"
 #include "core/PageTable.hpp"
+#include "core/PageTableWalker.hpp"
 #include <bitset>
 #include "sparta/utils/SpartaTester.hpp"
 
 class AtlasTranslateTester
 {
+    bool writeToMemory(uint64_t pa, uint64_t val)
```

Comment:
You can use the `AtlasState` method to write to memory now. It's been modified to use the PA.